elements/rh-pagination/rh-pagination.css

```
@@ -13,6 +13,33 @@
   flex-wrap: wrap;
 }
 
+#container.dark {
+  --_stepper-bg-color: var(--rh-color-surface-dark-alt, #292929);
+  --rh-pagination-accent-color: var(--rh-color-interactive-blue-lightest, #b9dafc);
+  --rh-pagination-stepper-color: var(--rh-color-text-secondary-on-dark, #c7c7c7);
+  --_stepper-color-focus: var(--rh-color-interactive-blue-lightest, #b9dafc);
+  --rh-pagination-background-focused: var(--rh-color-gray-50, #707070);
```

```suggestion
  --rh-pagination-accent-color: var(--rh-color-interactive-blue-lightest, #b9dafc);
  --rh-pagination-background-focused: var(--rh-color-gray-50, #707070);
  --rh-pagination-stepper-color: var(--rh-color-text-secondary-on-dark, #c7c7c7);
  --_stepper-bg-color: var(--rh-color-surface-dark-alt, #292929);
  --_stepper-color-focus: var(--rh-color-interactive-blue-lightest, #b9dafc);
```
to keep public and 'private' tokens organized